config/config.go

```diff
@@ -105,6 +105,9 @@ type Config struct {
 
 	// SSO in settings for single-sign on
 	SSO SSOConfig `json:"sso,omitempty"`
+
+	// Secret name for token auth mode
+	TokenSecretName string `json:"tokenSecretName,omitempty"`
```

Comment:
I think this should be

```yaml
token:
  secretName: abc
```

The secret name should have a default value, e.g. `argo-auth-tokens`